requirements.txt

```diff
@@ -1,5 +1,9 @@
 asgiref==3.5.0
 Django==3.2
+django-heroku==0.3.1
+gunicorn==20.1.0
 Pillow==9.0.1
+psycopg2==2.9.3
 pytz==2022.1
 sqlparse==0.4.2
+whitenoise==6.0.0
```

Comment:
whitenoise is for making it easier to deal with the static files?

Reply:
Yes, WhiteNoise allows web app to serve its own static files, making it a self-contained unit that can be deployed anywhere without relying on nginx, Amazon S3 or any other external service.